io.github.Torrent-file-editor.yaml

```diff
@@ -0,0 +1,39 @@
+app-id: io.github.torrent-file-editor
```

Comment:
this app-id doesn't meet the requirements.

Reply:
see https://docs.flatpak.org/en/latest/conventions.html#application-ids

Reply:
Fixed I also renamed the yaml file